### PR TITLE
Multi Module example

### PR DESCRIPTION
**Pull request by tramalho**
Multi module version based in mixed-languages

### CODE REVIEW

**Comment by rafaeltoledo** [comment]
I believe that Sonar configuration is a bit "out of scope" for this repository. Can you please remove the Sonar related files?

**Comment by tramalho** [comment]
Remove sonar references
Resolve marge conflicts
Rename package

File: another_module/src/test/java/net/rafaeltoledo/coverage/another/ExampleUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package net.rafaeltoledo.coverage.another;
```
Comment (rafaeltoledo) [inline another_module/src/test/java/net/rafaeltoledo/coverage/another/ExampleUnitTest.java]: Remove placeholder test

Comment (tramalho) [reply another_module/src/test/java/net/rafaeltoledo/coverage/another/ExampleUnitTest.java]: done

**Pull request merged**